argusclient/client.py

```diff
@@ -897,7 +932,7 @@ def logout(self):
         Logs out of the Argus service and destroys the session.
         """
         # The new V2 auth doesn't support a logout, so just clear the tokens.
-        #self._request("get", "auth/logout")
+        # self._request("get", "auth/logout")
```

Comment:
This is actually undesirable. I avoid putting a space when commenting out code as it helps me visually distinguish that from real comments that are used as documentation.